### PR TITLE
Update create-a-phone-system-call-queue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Updated information for External PSTN Transfers on Overflow and Timeout

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-call-queue.md [hunk 1]
```diff
@@ -148,6 +148,10 @@ You can choose to disconnect the call or redirect it to one of the call routing
 
 When you have selected your call timeout options, click **Save**.
 
+## Overflow and Timeout External PSTN Transfer Requirements
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:151]:
```suggestion
## Overflow and timeout external PSTN transfer requirements
```

File: Teams/create-a-phone-system-call-queue.md [hunk 2]
```diff
@@ -148,6 +148,10 @@ You can choose to disconnect the call or redirect it to one of the call routing
...
+
+Please refer to [External Phone Number Transfers](create-a-phone-system-auto-attendant.md#external-phone-number-transfers---technical-details)
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:153]:
```suggestion
Please refer to [External phone number transfers](create-a-phone-system-auto-attendant.md#external-phone-number-transfers---technical-details).
```

Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:153]:
```suggestion
Please refer to [External phone number transfers - technical details](create-a-phone-system-auto-attendant.md#external-phone-number-transfers---technical-details).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
This info has already been included via another PR.

**Pull request closed without merging**